cmds/install.go

```diff
@@ -50,7 +50,7 @@ const (
 	kubectl                  = "kubectl"
 	kubernetes               = "kubernetes"
 	oc                       = "oc"
-	binLocation              = "/.fabric8/bin/"
+	binLocation              = ".fabric8/bin/"
```

Comment:
You'll want to use filepath.Join in here too probably: `filepath.Join(".fabric8", "bin")`